File: app/src/main/java/com/decode/tumblr/data/model/PhotoObject.java
package com.decode.tumblr.data.model;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "photo_table")
public class PhotoObject {

    @PrimaryKey(autoGenerate = true)
    private int id;

    @NonNull
    private String url;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    @NonNull
    public String getUrl() {
        return url;
    }

    public void setUrl(@NonNull String url) {
        this.url = url;
    }

}

File: app/src/main/java/com/decode/tumblr/data/db/model/PhotoObject.kt
package com.decode.tumblr.data.db.model

import android.os.Parcelable
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.android.parcel.Parcelize

@Entity(tableName = "photo_table")
@Parcelize
data class PhotoObject(

        @PrimaryKey(autoGenerate = true)
        var id: Int = 0,

        var url: String = ""

) : Parcelable
